spec/controllers/verify/address_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Verify::AddressController do
+  def stub_subject(user)
+    user_session = {}
+    stub_sign_in(user)
+    idv_session = Idv::Session.new(user_session, user)
+    idv_session.resolution = Proofer::Resolution.new success: true, session_id: 'some-id'
+    idv_session.applicant = Proofer::Applicant.new first_name: 'Some', last_name: 'One'
+    idv_session.vendor = subject.idv_vendor.pick
+    allow(subject).to receive(:confirm_idv_session_started).and_return(true)
+    allow(subject).to receive(:confirm_idv_attempts_allowed).and_return(true)
+    allow(subject).to receive(:idv_session).and_return(idv_session)
+    allow(subject).to receive(:user_session).and_return(user_session)
```

Comment:
We typically don't have both controller and feature specs for the same functionality. In general, we favor controller specs because they are faster and because not every path needs to be covered by a feature spec. In some cases, such as when testing a flow through multiple controllers, a feature spec is preferred. If everything this controller does is already covered by feature specs, I would vote to remove this test. Anyone else have thoughts?